doctrine/mongodb-odm-bundle/3.3/etc/packages/doctrine_mongodb.yaml

```diff
@@ -0,0 +1,18 @@
+doctrine_mongodb:
+    auto_generate_proxy_classes: "%kernel.debug%"
+    auto_generate_hydrator_classes: "%kernel.debug%"
```

Comment:
In the officiel recipe for `doctrine/doctrine-bundle`, we use double quotes for this parameter.

See https://github.com/symfony/recipes/blob/master/doctrine/doctrine-bundle/1.6/etc/packages/doctrine.yaml#L5.